apps/antalmanac/src/components/Map/Map.tsx

```diff
@@ -217,7 +217,7 @@ export default function CourseMap() {
         <Box sx={{ width: '100%', display: 'flex', flexDirection: 'column', flexGrow: 1, height: '100%' }}>
             <MapContainer ref={map} center={[33.6459, -117.842717]} zoom={16} style={{ height: '100%' }}>
                 {/* Menu floats above the map. */}
-                <Paper sx={{ zIndex: 400, position: 'relative', my: 2, mx: 6.942, marginX: '15%', marginY: 8 }}>
+                <Paper sx={{ zIndex: 400, position: 'relative', my: 2, mx: 8 }}>
```

Comment:
```suggestion
                <Paper sx={{ zIndex: 400, position: 'relative', my: 2, mx: 16 }}>
```

Reply:
Alternatively, keeping the percentage-based MX would probably (?) be fine

Reply:
I prefer using rem whenever possible. 1 rem usually translates to 4px depending on the CSS framework's settings (it varies natively).